helpers/helpers.html

```diff
@@ -85,19 +85,20 @@
         var deltaScrollTop = scrollTop - currentScrollTop;
         var deltaScrollLeft = scrollLeft - currentScrollLeft;
         var duration = 300;
-
-        (function updateFrame() {
+        var updateFrame = (function updateFrame() {
```

Comment:
I figured it was for debugging, but the variable should probably take a different name so it's less ambiguous:

```javascript
var boundUpdateFrame = (function updateFrame() { ... }).bind(this);
```